cmd/flags.go

```diff
@@ -144,6 +145,12 @@ func addAudienceFlag(cmd *cobra.Command) {
 		"the audience field in the claims token. Cannot be sts.googleapis.com.")
 }
 
+// Lets this command specify custom nonce field of the attestation token.
+func addCustomNonceFlag(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringArrayVar(&customNonce, "custom-nonce", nil,
+		"the custom nonce field in the claims token. use this flag multiple times to add multiple custom nonce.")
```

Comment:
"the custom nonce field in the claims token. Use this flag multiple times to add multiple custom nonces."